### PR TITLE
bootstrap_sdk: enable arm64 sdk bootstrap

### PR DESCRIPTION
**Pull request by jepio**
# enable arm64 sdk bootstrap

Some issues with lib(64) handling and rust were found during attempts to bootstrap the SDK on arm64. This PR resolves them.

## How to use

Provision powerful arm64 machine.
Fetch kinvolk/coreos-overlay/pull/1130
Fetch stage4 from https://github.com/kinvolk/Flatcar/issues/319#issuecomment-881553797.
`./bootstrap_sdk --seed_tarball`

### CODE REVIEW

**Comment by jepio** [comment]
CI running http://jenkins.infra.kinvolk.io:8080/job/os/job/manifest/3139/

This change is actually independent of arm64 and is also needed for the profile upgrade (17.1) so it would be nice to get it merged soonish. It doesn't change anything functionally right now.

File: build_library/toolchain_util.sh [hunk 1]
```diff
@@ -333,7 +337,7 @@ install_cross_toolchain() {
         $sudo emerge "${emerge_flags[@]}" \
             "cross-${cross_chost}/gdb" "${cross_pkgs[@]}"
         if [ "${cross_chost}" = aarch64-cros-linux-gnu ] && \
```
Comment (pothos) [inline build_library/toolchain_util.sh:339]: Is the `cbuild` check needed here, too?

Comment (jepio) [reply build_library/toolchain_util.sh]: I added the cbuild check here for consistency.

**Review by pothos — APPROVED** [review]

**Review by dongsupark — COMMENTED** [review]
As I said in the meeting today, I managed to build an arm64 SDK from the PRs. 🎉 
Though I am not sure when we should merge the PRs, as we could start building another set of releases soon.
I will let @sayanchowdhury share some thoughts.

**Comment by jepio** [comment]
Finished a full amd64 bootstrap with this and all looks good. Talked to @sayanchowdhury and told him about merging this before release.

**Pull request merged**